test/clj/game/cards/operations_test.clj

```diff
@@ -2829,25 +2829,26 @@
       (click-prompt state :corp "Gain 2 [Credits]")
       (is (= 7 (:credit (get-corp))) "Corp gained 2 credits")))
   (testing "Purge virus counters, counts both Runner and Corp side (Sandstone)")
-  (do-game
-    (new-game {:corp {:deck ["Reverse Infection" "Sandstone"]}
-               :runner {:deck ["Virus Breeding Ground" "Datasucker" (qty "Sure Gamble" 4)]
-                        :hand ["Virus Breeding Ground" "Datasucker"]}})
-    (play-from-hand state :corp "Sandstone" "HQ")
-    (take-credits state :corp)
-    (play-from-hand state :runner "Virus Breeding Ground")
-    (play-from-hand state :runner "Datasucker")
-    (take-credits state :runner)
-    (core/add-counter state :corp (get-ice state :hq 0) :virus 3)
-    (core/add-counter state :runner (get-resource state 0) :virus 3)
-    (core/add-counter state :runner (get-program state 0) :virus 3)
-    (play-from-hand state :corp "Reverse Infection")
-    (click-prompt state :corp "Purge virus counters")
-    (is (= 7 (:credit (get-corp))) "Corp did not gain credits")
-    (is (zero? (get-counters (get-ice state :hq 0) :virus)) "Viruses purged from Sandstone")
-    (is (zero? (get-counters (get-resource state 0) :virus)) "Viruses purged from VBG")
-    (is (zero? (get-counters (get-program state 0) :virus)) "Viruses purged from Datasucker")
-    (is (= 3 (count (:discard (get-runner)))) "Three cards trashed from stack")))
+    (do-game
+      (new-game {:corp {:deck ["Reverse Infection" "Sandstone"]}
+                :runner {:deck ["Virus Breeding Ground" "Datasucker" (qty "Sure Gamble" 4)]
+                          :hand ["Virus Breeding Ground" "Datasucker"]}})
+      (play-from-hand state :corp "Sandstone" "HQ")
+      (take-credits state :corp)
+      (play-from-hand state :runner "Virus Breeding Ground")
+      (play-from-hand state :runner "Datasucker")
+      (take-credits state :runner)
+      (core/add-counter state :corp (get-ice state :hq 0) :virus 3)
+      (core/add-counter state :runner (get-resource state 0) :virus 3)
+      (core/add-counter state :runner (get-program state 0) :virus 3)
+      (play-from-hand state :corp "Reverse Infection")
+      (click-prompt state :corp "Purge virus counters")
+      (is (last-log-contains? state "Corp uses Reverse Infection to purge 9 virus counters and trash 3 cards from the top of the stack.") "Should write correct log")
+      (is (= 7 (:credit (get-corp))) "Corp did not gain credits")
```

Comment:
Can you remove this and the later lines? This test is only for checking the log, so we shouldn't recheck all of the previous test's assertions